src/compiler/crystal/tools/doc/html/404.html

```diff
@@ -0,0 +1,7 @@
+<h1 class="type-name">
+  404 Not Found
+</h1>
+
+<p>
+  This page is unavailable in this version of the API docs.
+</p>
```

Comment:
```suggestion
</p>

<p>
  You can use the sidebar to search for your page, or try a different
  Crystal version.
</p>
```
<img width="1025" alt="Screen Shot 2021-11-10 at 12 10 47" src="https://user-images.githubusercontent.com/5470539/141139014-0f36af6c-a44c-43f2-8fe4-814198ea9d51.png">